x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/analysis/VerifierTests.java

```diff
@@ -2253,6 +2253,21 @@ public void testLookupJoinExpressionAmbiguousRight() {
         );
     }
 
+    public void testLookupJoinExpressionRightNotPushable() {
+        assumeTrue("requires LOOKUP JOIN capability", EsqlCapabilities.Cap.JOIN_LOOKUP_V12.isEnabled());
```

Comment:
This cap is permanently enabled, no need to assume it.

(The reason that csv tests still require it is that queries with lookup joins are disabled in csv tests, but should run in EsqlSpecITs.)